qiita_db/software.py

```diff
@@ -508,6 +515,22 @@ def activate(self):
             qdb.sql_connection.TRN.add(sql, [True, self.id])
             return qdb.sql_connection.TRN.execute()
 
+    @property
+    def analysis_only(self):
+        """Returns if the command is an analysis-only command
```

Comment:
There are currently two options:
(1) The command is available in the analysis pipeline and the study processing pipeline
(2) The command is available only in the analysis pipeline.

I don't think the option "Only on study processing pipeline" should be added as this can potentially limit the opportunities to run meta-analysis, see my previous comment.